easybuild/easyblocks/l/llvm.py

```diff
@@ -104,23 +103,19 @@ def configure_step(self):
 
         if LooseVersion(self.version) >= LooseVersion('15.0'):
             # make sure that CMake modules are available in build directory,
-            # and if so make a 'cmake' symlink so LLVM can find them
+            # by moving the extracted folder to the expected location
             cmake_modules_path = os.path.join(self.builddir, 'cmake-%s.src' % self.version)
             if os.path.exists(cmake_modules_path):
-                cwd = change_dir(self.builddir)
-                symlink('cmake-%s.src' % self.version, 'cmake')
-                change_dir(cwd)
+                os.rename(cmake_modules_path, os.path.join(self.builddir, 'cmake'))
             else:
                 raise EasyBuildError("Failed to find unpacked CMake modules directory at %s", cmake_modules_path)
 
         if LooseVersion(self.version) >= LooseVersion('16.0'):
             # make sure that third-party modules are available in build directory,
-            # and if so make a 'third-party' symlink so LLVM can find them
+            # by moving the extracted folder to the expected location
             third_party_modules_path = os.path.join(self.builddir, 'third-party-%s.src' % self.version)
             if os.path.exists(third_party_modules_path):
-                cwd = change_dir(self.builddir)
-                symlink('third-party-%s.src' % self.version, 'third-party')
-                change_dir(cwd)
+                os.rename(third_party_modules_path, os.path.join(self.builddir, 'third-party'))
```

Comment:
we should prefer the `move_file` function provided by `tools.filetools`, I'll update this PR accordingly and then test with a bunch of LLVM easyconfigs